apps/website/next-sitemap.config.js

```diff
@@ -1,10 +1,18 @@
 // @ts-check
-/** @type {import('next-sitemap').IConfig} */
-module.exports = {
+
+const siteUrl = function () {
   // If there is a NEXT_PUBLIC_VERCEL_URL set, use that like NextAuth.js does
-  siteUrl: process.env.NEXT_PUBLIC_VERCEL_URL
+  const url = process.env.NEXT_PUBLIC_VERCEL_URL
     ? `https://${process.env.NEXT_PUBLIC_VERCEL_URL}`
-    : process.env.NEXT_PUBLIC_BASE_URL,
+    : process.env.NEXT_PUBLIC_BASE_URL;
+
+  //Fallback to localhost if no public url is present
+  return url ? url : "http://localhost";
+};
```

Comment:
GitHub won't let me apply a suggestion here, but I'm not sure I follow why this needs to be a method that gets called?

```js
// If there is a NEXT_PUBLIC_VERCEL_URL set, use that like NextAuth.js does
// Fallback to localhost if no public url is present
const siteUrl =  (process.env.NEXT_PUBLIC_VERCEL_URL
  ? `https://${process.env.NEXT_PUBLIC_VERCEL_URL}`
  : process.env.NEXT_PUBLIC_BASE_URL) || "http://localhost";
```